WORKSPACE

```diff
@@ -4,8 +4,24 @@ load(
     "@d2l_rules_csharp//csharp:defs.bzl",
     "csharp_register_toolchains",
     "csharp_repositories",
+    "import_nuget_package",
 )
 
 csharp_repositories()
 
 csharp_register_toolchains()
+
+# These are some examples.
+# Should this go here? Should we have a WORKSPACE inside csharp/ and one in
+# examples/?
```

Comment:
Maybe just in examples?

Reply:
Ya probably. I'll open an issue to do that tomorrow. It's going to be easier to configure Bazel CI with one WORKSPACE to start though so I'd like to get that done first so that I can test the change in CI. I'm happy to delay this change until that's set up though.